db/migrations/2025_04_03_125243_add_timestamps_to_study_and_project.py

```diff
@@ -0,0 +1,28 @@
+import sqlalchemy as sql
+
+
+def up(conn):
+    for table in ('Project', 'Study'):
+        query = f"""
+            ALTER TABLE {table}
+            ADD COLUMN createdAt datetime NOT NULL DEFAULT CURRENT_TIMESTAMP
+        """
+        conn.execute(sql.text(query))
+
+        query = f"""
+            ALTER TABLE {table}
+            ADD COLUMN updatedAt datetime NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
+        """
+        conn.execute(sql.text(query))
```

Comment:
Most records should have created/updated timestamps, in this particular PR, I needed to check if and when a particular study was being updated.